### ISSUE TITLE
Latex not rendering properly

### ISSUE THREAD

**Issue by rvzaytsev** [oleeskild/obsidian-digital-garden#551]
For some reason the latex equations are not rendered properly within ordered lists (and for that matter, the ordered list is also screwed up: no new lines between new numbers). See for example [my site](https://digital-garden-phi-lovat.vercel.app/), subsection "Proof" of the section "Inflection points".

**Comment by dayne** [comment]
The website is rendered using the [Eleventy](https://www.11ty.dev/) static site generator.  

So the question kind of becomes where do you debug and try to solve this.  On the Obsidian side or the 11ty side. 

If it is the 11ty side you'll have to dig into what plugins might take latex equations in markdown and render them correctly. 

I'm not sure what the right answer -- perhaps 
- provide expanded examples/context for the raw markdown on your obsidian vault that works
- along with what plugins are needed to make that markdown work in Obsidian.  
- Perhaps review to see what the ecosystem for 11ty for rendering Latex equations in Markdown.  My quick scan shows the answer might be [MathJax](https://www.mathjax.org/) because I see there is a [eleventy-plugin-mathjax](https://www.npmjs.com/package/eleventy-plugin-mathjax) available.  
   _warning: I have no clue how hard it is to enable MathJax on an Obsidian Digital Garden website._

**Comment by stalegjelsten** [comment]
@rvzaytsev, equations inside of ordered lists are working for me (I even tried with the list that didn't work for you). I suggest you take the troublesome list into a new file and check if it works OK there. 

Additionally, it seems the heading **Proof**, just above the messed up ordered list is not recognized as a heading. Did you make it a H4 just like all the other **Proof** headings? Could there be some formatting problem in the markdown file?

**Comment by rvzaytsev** [comment]
@stalegjelsten thanks, I've been able to get it working! It seems that the problem was in the fact that Latex was immediately after an Escalidraw sketch (in fact, I now noticed that it did not render in another place that did not have a numbered list). Adding a newline after Escalidraw sketch resolved the problem.
TL;DR: add a newline before something that contains latex.
